hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/container/ozoneimpl/TestOzoneContainerWithTLS.java

```diff
@@ -221,6 +229,67 @@ public void downloadContainer(boolean containerTokenEnabled)
       if (client != null) {
         clientManager.releaseClient(client, true);
       }
+      IOUtils.closeQuietly(clientManager);
+    }
+  }
+
+  @ParameterizedTest(name = "Container token enabled: {0}")
+  @ValueSource(booleans = {false, true})
+  public void testDNContainerOperationClient(boolean containerTokenEnabled)
+      throws Exception {
+    conf.setBoolean(HddsConfigKeys.HDDS_CONTAINER_TOKEN_ENABLED,
+        containerTokenEnabled);
+    OzoneContainer container = createAndStartOzoneContainerInstance();
+    DNContainerOperationClient dnClient = null;
+    XceiverClientSpi client = null;
+    try {
+      dnClient = new DNContainerOperationClient(conf, caClient, keyClient);
+      client = dnClient.getXceiverClientManager().acquireClient(pipeline);
+      long containerId = createAndCloseContainer(client, containerTokenEnabled);
+      dnClient.getContainerMerkleTree(containerId, dn);
+    } finally {
+      if (container != null) {
+        container.stop();
+      }
+      if (client != null) {
+        dnClient.getXceiverClientManager().releaseClient(client, true);
```

Comment:
Can we auto close the `DNContainerOperationClient` instead of manually releasing the clients?